src/core/inlines.js

```diff
@@ -191,7 +205,7 @@ export function run(conf) {
       "(?:{{[^}]+}})", // inline IDL references,
       "\\B\\|\\w[\\w\\s]*(?:\\s*\\:[\\w\\s&;<>]+)?\\|\\B", // inline variable regex
       "(?:\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\])",
-      "(?:\\[\\[\\[(?:!|\\\\|\\?)?[A-Za-z0-9\\.-]+\\]\\]\\])",
+      "(?:\\[\\[\\[(?:!|\\\\|\\?)?[#A-Za-z0-9\\.-]+\\]\\]\\])",
```

Comment:
I think we don't want to support `foo#bar` here, so it should be:
```suggestion
      "(?:\\[\\[\\[(?:!|\\\\|\\?)?#[A-Za-z0-9\\.-]+\\]\\]\\])",
```